platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/DataFileCollectionBench.java

```diff
@@ -60,7 +60,7 @@ BenchmarkRecord read(long dataLocation) throws IOException {
                 index.put(id, store.storeDataItem(record::serialize, BenchmarkRecord.getSerializedSize()));
                 if (verify) map[(int) id] = record;
             }
-            store.endWriting(0, maxKey).setFileCompleted();
+            store.endWriting(0, maxKey);
```

Comment:
I am not sure about tests and benchmarks, but in the main code `DataFileCollection.endWriting()` is always followed by `setFileCompleted()`. Why not call `setFileCompleted()` in the end of `endWriting()` and update the JavaDoc accordingly?